pkgs/by-name/de/dediprog-sf100/package.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, libusb
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "dediprog-sf100-linux";
+  version = "1.14.20.x";
+
+  src = fetchFromGitHub {
+    owner = "DediProgSW";
+    repo = "SF100Linux";
+    rev = "V${version}";
+    hash = "sha256-hQvBZIwaWEC41vj2flaekIUP9Fwtj/JPi3XwRxfUbD0=";
+  };
+
+  buildInputs = [ libusb ];
```

you could think about changing it to:
```suggestion
  buildInputs = [ libusb1 ];
```

---

Sounds good, as libusb1 is the currently used version.